src/Database/LibSQLDatabase.php

```diff
@@ -9,46 +9,37 @@ class LibSQLDatabase
 {
     protected LibSQL $db;
 
-    protected array $config;
-
     protected string $connection_mode;
 
     protected array $lastInsertIds = [];
 
     protected bool $inTransaction = false;
 
-    public function __construct(array $config = [])
-    {
-        $config = config('database.connections.libsql');
-
-        $url = str_replace('file:', '', $config['url']);
-        $config['url'] = match ($this->checkPathOrFilename($config['url'])) {
-            'filename' => 'file:'.database_path($url),
-            default => $config['url'],
-        };
-
-        $this->setConnectionMode($config['url'], $config['syncUrl'], $config['authToken'], $config['remoteOnly']);
+    public function __construct(protected array $config = []) {}
 
+    public function init(): void
```

Comment:
@darkterminal https://github.com/tursodatabase/turso-driver-laravel/pull/5#discussion_r1669040904
As i explained here, I don't understand why we should implement some custom "hacky" logic, instead of usage of built in Laravel logic of connection creation.

Logic you currently have doesn't work with multiple DB connections or connections defined not in `config('database.connections.libsql')`

You can pull this branch, add your changes and run tests to observe problems I'm talking about.